app/src/test/java/mozilla/lockbox/store/DataStoreTest.kt

```diff
@@ -58,7 +58,7 @@ class DataStoreTest : DisposingTest() {
         subject.unlock().subscribe(waiter)
         waiter.assertComplete()
 
-        waiter = createTestObserver<Unit>()
+        waiter = createTestObserver()
```

Comment:
nice.

Reply:
>nice.

All credit goes to the Android linter ❇️